src/components/User/User.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './User.scss';
+
+export const User = ({ name, email, address }) => (
+  <>
```

Comment:
you don't need this fragment, you already have div.user-info that wraps other elements